docs/api.rst

```diff
@@ -2,43 +2,55 @@ API Documentation
 =================
 
 
-Messages
+TChannel
 --------
 
-.. automodule:: tchannel.messages
+.. automodule:: tchannel.tornado.tchannel
     :members:
-    :imported-members:
 
 
-Framing
+Request Dispatch
+----------------
+.. automodule:: tchannel.tornado.dispatch
+    :members:
+
+
+Request
 -------
 
-.. automodule:: tchannel.frame
+.. automodule:: tchannel.tornado.request
     :members:
 
 
-Exceptions
-----------
+Response
+--------
 
-.. automodule:: tchannel.errors
+.. automodule:: tchannel.tornado.response
     :members:
 
-Connection
+
+Exceptions
 ----------
 
-.. automodule:: tchannel.tornado.connection
+.. automodule:: tchannel.errors
     :members:
 
 
-Request
--------
+Thrift
+------
 
-.. automodule:: tchannel.tornado.request
+.. automodule:: tchannel.thrift.client
+    :members:
+
+.. automodule:: tchannel.thrift.server
```

Comment:
Just the client should be fine. Functionality from `tchannel.thrift.server` is made available via the standard TChannel class.

Also, we should move this to `tchannel.tornado.thrift` at some point. We'll have to do it in a backwards compatible way (by keeping the old module around and re-exporting the new one from it).